### ISSUE TITLE
International travel/graduated

### ISSUE THREAD

**Issue by cdeboever3** [frazer-lab/cluster#136]
Hi @tatarsky, I wanted to let you know that I've graduated but am still using the cluster as we finish my last project and I contribute to some other projects. I'm not sure what your process is for retired users though I will still need access to the cluster for a while and @kafrazer is aware of this. I'll work on cleaning up my files over the next few months as my projects wind down. On a related note, I'm traveling internationally for the next three months, so if you see traffic from Thailand, Cambodia, or Europe, that's probably me. Thanks!

**Comment by tatarsky** [comment]
First off: Congratulations! Well done! I am always impressed with those that obtain additional degrees. After the drubbing I took at HMC I went right into industry to recover and have never considered such a thing. 

I will note that you are coming from various interesting IP ranges and not freak out and kill your shells. Thank you for all your help and effort and good luck in whatever comes next. 

I will discuss in general account policy next call. I think we need a catch up anyway as we've got a few issues we could tackle this summer. I'll gather them up but enjoy your trip!
